benchmark/framework/framework.go

```diff
@@ -77,7 +77,11 @@ func (frame *BenchmarkFramework) Run(ctx context.Context) {
 		}
 		testDriver.calculateStats()
 		if testDriver.AfterFunction != nil {
-			testDriver.AfterFunction()
+			err := testDriver.AfterFunction()
+			if err != nil {
+				fmt.Printf("File Access Parsing Error: %v\n", err)
```

Comment:
AfterFunction could be anything. Don't mention "File Access Parsing Error".

Reply:
Got it will remove it.